File: Python/ex4.py
#Escreva um algoritmo que receba a idade e o peso de sete pessoas. Calcule e exiba a quantidade de pessoas 
#com mais de 90 quilos. A média das idades das sete pessoas.

count_acima_90kg = 0
idade_total = 0
for i in range(7):
    idade = int(input(f"Digite a idade da {i+1}ª pessoa: "))
    peso = float(input(f"Digite o peso da {i+1}ª pessoa (em kg): "))
    if peso > 90:
        count_acima_90kg += 1
    idade_total += idade
print("Quantidade de pessoas com mais de 90 quilos:", count_acima_90kg)
print("Média das idades:", idade_total / 7)

File: Java/ex4.java
// Escreva um algoritmo que receba a idade e o peso de sete pessoas. Calcule e exiba a quantidade de pessoas 
//com mais de 90 quilos. A média das idades das sete pessoas.

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int peso90 = 0;
        int somaIdades = 0;

        for (int i = 0; i < 7; i++) {
            System.out.println("Digite a idade:");
            int idade = scanner.nextInt();
            System.out.println("Digite o peso:");
            float peso = scanner.nextFloat();

            if (peso > 90) {
                peso90++;
            }
            somaIdades += idade;
        }

        System.out.println("Pessoas com mais de 90 quilos: " + peso90);
        System.out.println("Média das idades: " + (somaIdades / 7.0));
    }
}
